GETTING_STARTED.md

```diff
@@ -33,6 +33,34 @@ import { MaterialModule } from '@angular/material';
 export class PizzaPartyAppModule { }
 ```
 
+## Tell to System loader where Angular Material is located
+This is **required** step. You need to tell to system loader where angular material is located otherwise
```

Comment:
Configuring SystemJS is only necessary _if you're using SystemJS_. If you're using Webpack  (such as with angular-cli) or something like Browserify, RequireJS etc., this isn't necessary. 

I'm okay with adding something on SystemJS since the angular quickstart uses it, but it should be further down (since not _all_ projects use it). I'd put it as something like:
```md
## Configuring SystemJS
If your project is using SystemJS for module loading, you will need to add `@angular/material` to the SystemJS configuration:
System.config({
  // existing configuration options
  map: {
    ...,
    '@angular/material': 'npm:@angular/material/material.umd.js'
  }
});
```